packages/ladle/lib/shared/types.ts

```diff
@@ -130,8 +130,10 @@ export type StoryProps = {
   config: Config;
 };
 
+/* P is unused but left there for backwards compatibility */
```

Comment:
Technically this change is 10 hours old, I'll leave it for you consideration @tajo.